test/Tereshchenko_Igor/laboratory1.sql

```diff
@@ -1,2 +1,152 @@
 -- LABORATORY WORK 1
 -- BY Tereshchenko_Igor
```

Comment:
Delete this
` -- LABORATORY WORK 1
 -- BY Tereshchenko_Igor`

Reply:
Fixed